studio/src/hooks/use-hydrate-playground-state-from-url.ts

```
@@ -87,13 +95,16 @@ export const useHydratePlaygroundStateFromUrl = (
       });
       setIsHydrated(true);
     } finally {
+      if (!playgroundUrlState) {
+        return;
```

Biome (2.4.15)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)